src/state/opamState.ml

```diff
@@ -1292,6 +1297,24 @@ let dump_state t oc =
         (string_of_conjunction (fun a -> a) m);
     with Not_found -> () );
 
+    (match repository_and_prefix_of_package t package with
```

Comment:
yup ! I didn't understand the interface : too many functions with a similar name... much easier this way.